org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -470,6 +470,38 @@ GithubOidcSageBionetworksSynapseR:
     Repositories:
       - name: "synapser"
         branches: ["*"]
+      - name: "synapserutils"
```

Comment:
I wasn't sure if this would be better...

Reply:
@brucehoff just tagging you here to make sure this isn't missed. There's two ways to do this, just wanted to check it there was a preferable method.

Reply:
either option would work but you only need one of them.  since it's the same policy going with this one is the better approach.  BTW, to be more secure you can also specify specific branches, for example:  `branches: ["master","dev", "prod", "etc"]`

Reply:
We are using tags to release, so let's keep as this for now.

Reply:
By default we allow  all tags to access accounts @thomasyu888.  Think of it as `tags: ["*"]`  always.

Reply:
Thanks @zaro0508 - i did not know that!  can you do a re-review?  I made modifications!